src/pages/scheduler/components/CourseCard.tsx

```diff
@@ -68,14 +88,29 @@ export function CourseCard({
           </Flex>
         </Flex>
         <Flex direction="row" alignItems="center" justifyContent="space-between" w="100%">
-          <VStack alignItems="start" spacing="0" py="2">
-            <Text fontSize="lg" fontWeight="bold">
-              {subject} {code}
-            </Text>
-            <Text fontSize="sm" fontWeight="normal">
-              <Skeleton isLoaded={!loading}>{data?.title ?? ''}</Skeleton>
-            </Text>
-          </VStack>
+          <Flex grow={1}>
+            <VStack
+              alignItems="start"
+              spacing="0"
+              py="2"
+              tabIndex={0}
```

Comment:
I just tested this, it doesn't work when `tabIndex` is set on the VStack, but it does work if you set it on one of the `Text` children. Just setting it on one should be fine

Reply:
Moved tabIndex to just the course code and subject.